test/bash/bash.go

```diff
@@ -1,23 +1,47 @@
 package bash
```

Comment:
This package contains tools to run processes but not necessarily in bash (or any shell). Following unix standards, I'd name it `exec`.
```suggestion
package exec
```

Reply:
`exec` is kind of misleading, as non of the functions in there actually executes anything, it's just wrappers for string constructions for (shell) commands.

Reply:
Ok, `shell` is better than `bash`.
But commands don't need to be executed in bash, subprocess can just fork from current process and there is no shell underneath. This is why I'd suggest non-shell package name. How about `commands`?